src/main/java/sirius/biz/codelists/CodeListLookupTable.java

```diff
@@ -131,4 +131,9 @@ private boolean filter(CodeListEntry<?, ?> entry, String searchTerm, String lang
     protected Stream<LookupTableEntry> performScan(String lang) {
         return codeLists.getEntries(codeList).stream().map(entry -> extractEntryData(entry, lang));
     }
+
+    @Override
+    protected Stream<LookupTableEntry> performLookupScan(String lang, String lookupPath, String lookupValue) {
```

Comment:
Optional: "performQuery" - scan is used to indicate that "no filter is applied at all"